docs/scanning-for-extension-points.md

```diff
@@ -0,0 +1,40 @@
+# Scanning for Extension Points
+
+On initialization, essentially as soon as a FakeItEasy type is
+accessed, FakeItEasy uses reflection to look for internal and
+user-supplied extension points. In most cases, there is no _need_ for
+users to define any extensions, but they may be used to enhance the
+power and usability of FakeItEasy.
+
+There are currently three kinds of extension points defined:
+
+* [Custom Dummy Creation](custom-dummy-creation.md) rules,
+* Fake Configurators

```

Comment:
> If you'd like, I can write something up.

I don't think it's worth the trouble. Most users will probably want to use the 2.0 version anyway.